mod-vendors/examples/vendor_post.sample

```diff
@@ -69,63 +69,61 @@
-        "last_name": "Fury",
-        "language": "en-us",
-        "notes": "SHIELD's Big Kahuna",
-        "phone_numbers": [
-          {
-            "phone_number": "19789999999",
-            "categories": [
-              "0e3f9680-ab06-4565-af64-609b7364e6eb",
-              "996ecd31-7ca4-4d8d-9bbf-bc94dff5f6c6"
-            ],
-            "language": "en-us"
-          }
-        ],
-        "emails": [
-          {
-            "value": "noreply@folio.org",
-            "description": "Main",
-            "categories": [
-              "112ae2e4-88ae-4fa5-a75b-2379d2035e52",
-              "56288fe8-8037-44da-8395-01d2d106dc54"
-            ],
-            "language": "en-us"
-          }
-        ],
-        "addresses": [
-          {
-            "addressLine1": "10 Estes Street",
-            "addressLine2": "",
-            "city": "Ipswich",
-            "stateRegion": "MA",
-            "zipCode": "01938",
-            "country": "USA",
-            "categories": [
-              "0e3f9680-ab06-4565-af64-609b7364e6eb",
-              "996ecd31-7ca4-4d8d-9bbf-bc94dff5f6c6",
-              "112ae2e4-88ae-4fa5-a75b-2379d2035e52",
-              "56288fe8-8037-44da-8395-01d2d106dc54"
-            ],
-            "language": "en"
-          }
-        ],
-        "urls": [
-          {
-            "value": "noreply@folio.org",
-            "description": "corporate",
-            "language": "en-us",
-            "categories": [
-              "112ae2e4-88ae-4fa5-a75b-2379d2035e52",
-              "56288fe8-8037-44da-8395-01d2d106dc54"
-            ],
-            "notes": "note"
-          }
-        ]
-      },
+      "prefix": "Director",
+      "first_name": "Nick",
+      "last_name": "Fury",
+      "language": "en-us",
+      "notes": "SHIELD's Big Kahuna",
+      "phone_numbers": [
+        {
+          "phone_number": "19789999999",
+          "categories": [
+            "0e3f9680-ab06-4565-af64-609b7364e6eb",
+            "996ecd31-7ca4-4d8d-9bbf-bc94dff5f6c6"
+          ],
+          "language": "en-us"
+        }
+      ],
+      "emails": [
+        {
+          "value": "noreply@folio.org",
+          "description": "Main",
+          "categories": [
+            "112ae2e4-88ae-4fa5-a75b-2379d2035e52",
+            "56288fe8-8037-44da-8395-01d2d106dc54"
+          ],
+          "language": "en-us"
+        }
+      ],
+      "addresses": [
+        {
+          "addressLine1": "10 Estes Street",
+          "addressLine2": "",
+          "city": "Ipswich",
+          "stateRegion": "MA",
+          "zipCode": "01938",
+          "country": "USA",
+          "categories": [
+            "0e3f9680-ab06-4565-af64-609b7364e6eb",
+            "996ecd31-7ca4-4d8d-9bbf-bc94dff5f6c6",
+            "112ae2e4-88ae-4fa5-a75b-2379d2035e52",
+            "56288fe8-8037-44da-8395-01d2d106dc54"
+          ],
+          "language": "en"
+        }
+      ],
+      "urls": [
+        {
+          "value": "noreply@folio.org",
```

Comment:
is this a valid value for this field?

Reply:
Looks like it is given the current state of the schema since it will accept any string.  @komick @bonthebeerman do you guys think it's worth adding any restriction to this?  i.e. a pattern?

There are several degrees of strictness we could go with here... e.g. `^.+://.+$` being on the lenient end of the spectrum, and something like: `^(http|ftp)(s)?://.+$` being more restrictive as it only would allow for certain URI schemes/protocols

Reply:
Arvind and I discussed and agreed to go with ^(http|ftp)(s)?://.+$  Validation will be enforced on both the UI and backend.

Reply:
Also added url formatting to edi-ftp.server_address

Reply:
URL formatting regex removed and re-added after verifying that server_addresses in sample data rather than examples need updating.